### ISSUE TITLE
Add some more scenarios

### ISSUE THREAD

**Issue by TomasRoj** [danthareja/contribute-to-open-source-server#10]
Hey,

I think that only one story is not much for beginner. By adding more "stories" beginner can exercise more and make smaller mistakes in the future. If you want I can help.

Tom

**Comment by danthareja** [comment]
Thanks for the issue @TomasRoj. I would very much like to include some more advanced content, I just haven't had the time to build it yet.

I've found that the next big step for beginners is pulling changes from upstream and handling merge conflicts.

What do you think? How would you approach this?

**Comment by TomasRoj** [comment]
Yes it can be good example. If you want I can look at it with my friends and do that 😊

**Comment by danthareja** [comment]
I’m headed out of town for the next two weeks and won’t be online, but if you want to take lead on this I’ll be happy to review when I get back in town.

**Comment by TomasRoj** [comment]
Ok, IDK if I can get it to that time I will try it. Thanks for now
